example/lib/main.dart

```diff
@@ -201,6 +201,7 @@ class _MyHomePageState extends State<MyHomePage> {
                   selectedUrl,
                   rect: Rect.fromLTWH(0.0, 0.0, MediaQuery.of(context).size.width, 300.0),
                   userAgent: kAndroidUserAgent,
+                  invalidUrlRegex: r'^(https).+(twitter)', // prevent redirecting to twitter when user click on its icon in flutter website
```

Comment:
This code doesn't work for me on iOS simulator

Reply:
I still get it worked in my iOS simulator. Note that: `invalidUrlRegex` is only set up for WebView launched by 'Open Webview (rect)' button.

Reply:
Ah yes, you're right. It works now. Thank you!